_content/articles/spatiotemporal-concepts-of-nishida-kitaro-and-dogen_Raud-Rein.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "‘Place’ And ‘Being-Time’: Spatiotemporal Concepts In The Thought Of Nishida Kitaro And Dogen Kigen"
+authors:
+  - "Rein Raud" 
+external_url: "https://www.thezensite.com/ZenEssays/Philosophical/Place_and_BeingTime.html"
+drive_links:
+  - "https://drive.google.com/file/d/1FLRv2M_dSgTSE5BvvQCcOVYx_aEv08WH/view?usp=sharing"
+course: dialogue
+tags:
+  - philosophy
+  - west
+  - east-asian
+  - chan
+year: 2004
+month: jan
+journal: 
+volume: 54
+number: 1
+pages: "29--51" 
+---
+
+A comparative analysis of Kitarō Nishida and Dōgen Kigen's thoughts on space and time and how these concepts are presented throughout their life's work. The article largely focuses on the thought of Nishida, a 20th-century Japanese philosopher. While it is known that Nishida was greatly influenced by Western philosophy, the author brings Nishida into dialogue with Dōgen in an attempt to show that Nishida was firmly rooted in Asian thought.
```

Comment:
Let's add a quote above this, and definitely link to our entry for Dōgen's "Being Time": https://buddhistuniversity.net/content/essays/time-being_dogen